sdk/include/sdk/Exceptions.h

```diff
@@ -28,24 +27,13 @@ namespace velocitas {
  */
 class RpcException : public std::runtime_error {
 public:
-    explicit RpcException(const std::string& message)
-        : std::runtime_error(message) {}
+    using std::runtime_error::runtime_error;
     ~RpcException() override = default;
 
-    RpcException(const RpcException& other)
-        : std::runtime_error(other) {}
-    RpcException(RpcException&& other)
-        : std::runtime_error(other) {}
-
-    RpcException& operator=(const RpcException& other) {
-        std::runtime_error::operator=(other);
-        return *this;
-    }
-
-    RpcException& operator=(RpcException&& other) {
-        std::runtime_error::operator=(other);
-        return *this;
-    }
+    RpcException(const RpcException& other)            = default;
+    RpcException(RpcException&& other)                 = default;
+    RpcException& operator=(const RpcException& other) = default;
+    RpcException& operator=(RpcException&& other)      = default;
```

Comment:
Do we want to keep that (same for d-tor)? 
Makes it more explicit, but is also a bit "chatty".

Reply:
Wherever possible I would like to have as little extra code as possible. If we can remove it without any SCA complaining, lets go for it.

Reply:
Seems the SCAs of our current CI workflow do not complain. So, I will push it ...